<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Limits</class>
 <widget class="QWidget" name="Limits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limits</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="unitHead" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="unit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Air Quality COâ‚‚</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>280</width>
     <height>130</height>
    </rect>
   </property>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>border
in ppm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>difference
in ppm</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="value1">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>20</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>400.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1500.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>50.000000000000000</double>
    </property>
    <property name="value">
     <double>800.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="value2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>75</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>50.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="lower1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lower2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>75</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="raise1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="raise2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>75</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
